lang/en-gb.xml

```diff
@@ -580,7 +580,7 @@ After the download is complete, Vita3K will restart automatically and then insta
 		<later_version_already_installed>The later version of Vita3K is already installed.</later_version_already_installed>
 		<latest_version_already_installed>The latest version of Vita3K is already installed.</latest_version_already_installed>
 		<new_features>New Features in Version {}</new_features>
-		<authors>Authors</authors>
+		<authors>Author</authors>
```

Comment:
Revert

Reply:
The actual GUI for updating only shows one author, and "authors" is multiple, so it is correct for there to be no "s".

Reply:
Code show author for all commit, so if have few commit, have few authors

Reply:
oh right, didnt see that.

Reply:
fixed